src/components/header/examples/header.scss

```diff
@@ -1,7 +1,6 @@
 :host(limel-example-header) {
     --header-background-color: rgb(var(--contrast-1100));
     --header-heading-color: rgb(var(--contrast-100));
-    --header-subheading-color: rgb(var(--color-teal-lighter));
     --header-supporting-text-color: rgb(var(--color-red-lighter));
     --header-icon-color: rgb(var(--color-red-light));
 }
```

Comment:
This doesn't seem to have any effect anymore. 👇 Not 100% sure if it ever did.
````scss
limel-icon-button {
    color: rgb(var(--contrast-100));
}
````
We should probably add a new public variable to `limel-icon` (in another PR or commit). For example `--icon-fill-color`, and consequently add the same to variable to `limel-icon-button`, and use it here. If we really wan ti. 

Otherwise, it is better to remove it.